manifests/v/venko/ClassicGenerator/5.0.0.3/venko.ClassicGenerator.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: venko.ClassicGenerator
+PackageVersion: 5.0.0.3
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://cg.venko.me/pkg/ClassicGenerator-5.0.0.3-win.exe
+  InstallerSha256: C0D71A667DEC76495F1487098C5CCB8F11C0834A378ED6DF35B154B287F09FF7

```

```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://cg.venko.me/pkg/ClassicGenerator-5.0.0.3-win.exe
  InstallerSha256: C0D71A667DEC76495F1487098C5CCB8F11C0834A378ED6DF35B154B287F09FF7
  InstallerSwitches: 
    - Custom: /ALLUSERS
  Scope: machine
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://cg.venko.me/pkg/ClassicGenerator-5.0.0.3-win.exe
  InstallerSha256: C0D71A667DEC76495F1487098C5CCB8F11C0834A378ED6DF35B154B287F09FF7
  InstallerSwitches: 
    - Custom: /CURRENTUSER
  Scope: user
```
Switches are needed to install silently